packages/components/icons/package.json

```diff
@@ -22,10 +22,17 @@
       "./index.ts",
       "./custom-icon/index.ts",
       "./inline-svg/index.ts",
-      "./leading-icon/index.ts"
+      "./leading-icon/index.ts",
+      "./generated/*.tsx"
     ]
   },
-  "files": ["dist", "custom-icon", "inline-svg", "leading-icon"],
+  "files": [
+    "dist",
+    "custom-icon",
+    "inline-svg",
+    "leading-icon",
+    "generated/**"
```

Comment:
This glob tells preconstruct that we expect the dist/package.json for each icon to be in the `generated` folder.